File: hmcldryl/lib/models/BlogPost.dart
import 'package:cloud_firestore/cloud_firestore.dart';

class BlogPost {
  String title;
  String content;
  Timestamp timestamp;

  BlogPost(this.title, this.content, this.timestamp);

  BlogPost.fromSnapshot(snapshot)
      : title = snapshot.data()['title'],
        content = snapshot.data()['content'],
        timestamp = snapshot.data()['timestamp'];
}

File: android/app/src/main/java/com/hmcldryl/app/system/models/BlogPost.java
package com.hmcldryl.app.system.models;

import com.google.firebase.Timestamp;

public class BlogPost {

    String title,
    content;
    Timestamp timestamp;

    public BlogPost() {
    }

    public BlogPost(String title, String content, Timestamp timestamp) {
        this.title = title;
        this.content = content;
        this.timestamp = timestamp;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public Timestamp getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(Timestamp timestamp) {
        this.timestamp = timestamp;
    }
}
